### ISSUE TITLE
Show the number of commits

### ISSUE THREAD

**Issue by icyflame** [guanqun/git-punchcard-plot#9]
Have an option that would print the actual number of commits in a particular segment (such as Monday, 10 am - 11 am)

A bar graph visualisation for this might be better (although out of the scope of the present project)

**Comment by guanqun** [comment]
Hmm. Where to show the numbers properly? In the circle might not be appropriate because the numbers would be too small.

**Comment by icyflame** [comment]
Yeah, true. I don't know where. If we follow GitHub's punchcard, then it should be a tool tip on top of the circle.

**Comment by icyflame** [comment]
@guanqun I am going to try to do this using the `show_text` function. We can show the text at the center of the punch itself, but I have no clue how it will look! I will post an example soon.

**Comment by guanqun** [comment]
Sure. We can start from an example. :)

**Comment by icyflame** [comment]
![output](https://user-images.githubusercontent.com/3668034/31479668-40d263d4-af08-11e7-8d81-24dfe19955b7.png)

This seems to work. The 1 is still out of place, a good way to do this could be to find the font size which will lead the 1 to comfortably fit within the circle radius for 1 commit and then use that font size everywhere. That should solve our problem.

This SO answer has a way to do that, I can implement this. https://stackoverflow.com/a/2836530/2080089

What do you think?

**Comment by guanqun** [comment]
The graph looks great, however it seems to me the number is not strictly in the middle of the circle, slightly towards the right-bottom?

Using the method you mentioned can probably work. But what if the font size calculated is too small? We could have a default threshold, if it's too small, we can simply discard that number.
